mep/people/admin.py

```diff
@@ -535,12 +535,12 @@ def after_import(self, dataset, result, using_transactions, dry_run, **kwargs):
             if self.objects_to_index:
                 Person.index_items(self.objects_to_index)
 
-            # disconnect indexing once done
-            IndexableSignalHandler.disconnect()
-
         # turn viaf lookups back on
         settings.SKIP_VIAF_LOOKUP = False
 
+        # make sure indexing disconnected afterward
+        IndexableSignalHandler.disconnect()
+
```

Comment:
This in `after_import()` seems to fix all remaining errors.

Reply:
because after import is not run by the test code?

Reply:
It is. We just needed to make sure that the indexer is disconnected after we potentially use it in that same function. I thought it was best to leave it on but it's best to leave it off it seems. The other tests likely handle it on their own?